app/src/main/res/layout/item_chatroom_header.xml

```diff
@@ -14,14 +15,14 @@
         style="@style/ChatRooms.Header"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:padding="8dp"
+        android:padding="16dp"
         android:paddingEnd="@dimen/screen_edge_left_and_right_padding"
         android:paddingStart="@dimen/screen_edge_left_and_right_padding"
         android:text="@string/chatroom_header" />
 
-    <View
+    <!--<View
```

Comment:
Isn't it necessary? Then remove this commented code.

Reply:
> There is something to be improved by seeing the screenshot above. Note that a divider is being shown below the channels separator (sandbox channel).

@filipedelimabrito Yes, I had seen it too! I was thinking about opening a new issue, but with your suggestion I'll fix it in this PR. What do you think?

Reply:
It really needs to be fixed on this PR. Would be grate if you change that screen to what we have here 
https://github.com/RocketChat/Rocket.Chat.Android/pull/2110#issuecomment-470986869